examples/network-load-balancer/main.tf

```diff
@@ -0,0 +1,102 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# LAUNCH A NETWORK LOAD BALANCER
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  # The modules used in this example have been updated with 0.12 syntax, which means the example is no longer
+  # compatible with any versions below 0.12.
+  required_version = ">= 0.12"
+}
+
+# ------------------------------------------------------------------------------
+# CONFIGURE OUR GCP CONNECTION
+# ------------------------------------------------------------------------------
+
+provider "google" {
+  version = "~> 2.7.0"
+  region  = var.region
+  project = var.project
+}
+
+provider "google-beta" {
+  version = "~> 2.7.0"
+  region  = var.region
+  project = var.project
+}
+
+# ------------------------------------------------------------------------------
+# CREATE THE INTERNAL TCP LOAD BALANCER
+# ------------------------------------------------------------------------------
+
+module "lb" {
+  source = "../../modules/network-load-balancer"
```

Comment:
Should we have our usual git https ref boilerplate here?

Reply:
Will add